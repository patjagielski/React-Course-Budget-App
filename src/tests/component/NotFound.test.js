import { shallow } from'enzyme';
import React from 'react';
import NotFoundPage from '../../components/NotFound';

test('render the NotFoundPage correctly', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
});