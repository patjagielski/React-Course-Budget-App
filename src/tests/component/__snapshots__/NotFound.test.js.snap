// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the NotFoundPage correctly 1`] = `
<div>
  404 ~ Return to the homepage ~ 
  <Link
    replace={false}
    to="/"
  >
    Home
  </Link>
</div>
`;
